src/components/CaptureDetailDialog.js

```diff
@@ -265,14 +297,70 @@ function CaptureDetailDialog(props) {
           )}
 
           <Typography variant="subtitle1">Other</Typography>
-          {allTags.length === 0 ? (
+          {otherTags.length + mockCaptureTags.length === 0 ? (
             <Typography variant="body1">---</Typography>
           ) : (
-            <div className={classes.chipRoot}>
-              {allTags.map((tag) => (
-                <Chip key={tag} label={tag} className={classes.chip} />
-              ))}
-            </div>
+            <>
+              <div className={classes.chipRoot}>
+                {otherTags.map((tag) => (
+                  <Chip key={tag} label={tag} className={classes.chip} />
+                ))}
+
+                {mockCaptureTags.map((tag) => (
+                  <Chip
+                    key={tag.tagName}
+                    label={tag.tagName}
+                    className={classes.chip}
+                    onDelete={
+                      hasApproveTreePermission !== undefined // TODO: delete, for testing purpose only
+                        ? // onDelete={(hasApproveTreePermission
+                          () => {
+                            setCaptureTagDeletionTarget(tag.tagName);
+                          }
+                        : undefined
+                    }
+                  />
+                ))}
+              </div>
+
+              {captureTagDeletionTarget !== undefined && (
+                <Container
+                  style={{
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                    padding: '0px',
+                    marginTop: '1em',
+                  }}
+                >
+                  <Typography>
+                    Remove tag <b>{`"${captureTagDeletionTarget}"`}</b> ?
+                  </Typography>
+
+                  <ButtonGroup>
```

Comment:
I'd suggest using two separate buttons with a gap rather than a `ButtonGroup`. This looks more like a toggle switch, and no gap makes mis-clicks more likely.